src/vorta/store/models.py

```diff
@@ -97,6 +97,7 @@ class BackupProfileModel(BaseModel):
     pre_backup_cmd = pw.CharField(default='')
     post_backup_cmd = pw.CharField(default='')
     dont_run_on_metered_networks = pw.BooleanField(default=True)
+    allow_new_networks = pw.BooleanField(default=False)
```

Comment:
Here it defaults to `False`, but in migrations it's `True`? Suggest to use `True` everywhere, which is the current behavior.

Reply:
I think yf and I discussed this before (been long so idr exactly where, I tried searching). The new behaviour should not surprise existing users, so the migration changes nothing. But for new users, having this value as false is more beneficial.

Reply:
found it - https://github.com/borgbase/vorta/pull/1658#discussion_r1140417145